7-layouts/styles.css

```
@@ -0,0 +1,145 @@
+/*Mobile design*/
+.articles {
+    display: flex;
+    flex-flow: column wrap;
+    margin-top: 40px;
+}
+
+.articles .animal {
+    display: flex;
+    flex-direction: column-reverse;
+    height: auto;
+    margin-bottom: 35px;
+}
+
+.articles .animal h3 {
+    background-color: rgba(88, 88, 88, 0.281);
+    font-size: 25px;
+    font-weight: 10;
+    position: absolute;
+    margin: 0;
+    width: 100%;
+}
+
+.articles .animal .animal-img {
+    display: block;
+    height: auto;
+    width: 100%;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.header section:first-child {
```

```suggestion
.header .section:first-child {
```
Use classnames so your HTML doesn't have to match your styles.